consensus/reactor.go

```diff
@@ -177,6 +177,13 @@ func (conR *ConsensusReactor) AddPeer(peer p2p.Peer) {
 	}
 }
 
+func (conR *ConsensusReactor) InitAddPeer(peer p2p.Peer) p2p.Peer {
```

Comment:
Should we return a peer OR just state in the interface that InitAddPeer might modify the peer?

Reply:
I prefer return a peer, more literally, no not need explain anything.  What do you think?